pkg/transformer/utils.go

```diff
@@ -141,3 +143,80 @@ func Print(name, path string, trailing string, data []byte, toStdout, generateJS
 	}
 	return file, nil
 }
+
+// GetComposeFileDir returns compose file directory
+func GetComposeFileDir(inputFiles []string) (string, error) {
+	// Lets assume all the docker-compose files are in the same directory
+	inputFile := inputFiles[0]
+	if strings.Index(inputFile, "/") != 0 {
+		workDir, err := os.Getwd()
+		if err != nil {
+			return "", err
+		}
+		inputFile = filepath.Join(workDir, inputFile)
+	}
+	log.Debugf("Compose file dir: %s", filepath.Dir(inputFile))
+	return filepath.Dir(inputFile), nil
+}
+
+func BuildDockerImage(service kobject.ServiceConfig, name string, relativePath string) error {
+
+	// First, let's figure out the relative path of the Dockerfile!
+	// else, we error out.
+	if _, err := os.Stat(service.Build); err != nil {
+		return fmt.Errorf("%s is not a valid path for building image %s. Check if this dir exists.", service.Build, name)
```

Comment:
nit: how about `errors.Wrap`, since that is what we are using in rest of the code?